dev/README.md

```diff
@@ -2,8 +2,19 @@
 
 When you're developing your own app, you need an actual Mattermost server to be running. The Apps Framework development environment helps accomplish this by setting up a minimalistic environment with just two containers. One is for the database Mattermost communicates with, and the other container runs the actual Mattermost server. The other containers present in the Mattermost development environment are unnecessary for the purposes of building apps. So the advantage here is that there is just one dependency to start developing apps.
 
+Run the following commands in two different terminals to have your app run in its own terminal:
```

Comment:
Should doing this will open up a third terminal?

Reply:
@justinegeffen No the second terminal is the "extra" one in this case. Does that make sense?